lib/Suggestions.js

```diff
@@ -29,12 +50,12 @@ class Suggestions extends Component {
     const { props } = this;
     const suggestions = props.suggestions.map(function (item, i) {
       return (
-        <li key={i}
-        onMouseDown={props.handleClick.bind(null, i) }
-        onMouseOver={props.handleHover.bind(null, i) }
-        className={i == props.selectedIndex ? "active" : ""}>
-        <span dangerouslySetInnerHTML={this.markIt(item, props.query) } />
-        </li>
+          <li key={i}
+              onMouseDown={props.handleClick.bind(null, i) }
+              onMouseOver={props.handleHover.bind(null, i) }
+              className={i == props.selectedIndex ? "active" : ""}>
```

Comment:
Changes here should be considered in a different case, in my opinion, as this pull request focuses on the scrolling issue specifically.